servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -345,7 +345,50 @@ float sample_pcf_shadow(texture2D shadow, vec2 shadow_pixel_size, vec4 coord) {
 		avg += textureProj(sampler2DShadow(shadow, shadow_sampler), vec4(pos + shadow_pixel_size * (disk_rotation * scene_data.soft_shadow_kernel[i].xy), depth, 1.0));
 	}
 
-	return avg * (1.0 / float(sc_soft_shadow_samples));
+	return avg / float(sc_soft_shadow_samples);
```

Comment:
This should remain as it was before. My understanding of specialization constants is that they are available at compile time. Accordingly the ``(1.0 / float(sc_soft_shadow_samples))`` will resolve at compile time and the end result will be a single multiplication at run time instead of a division.

Reply:
Yup, you are right. Changed it back.